libs/community/langchain_community/document_loaders/parsers/grobid.py

```diff
@@ -48,7 +48,11 @@ def process_xml(
             )
         soup = BeautifulSoup(xml_data, "xml")
         sections = soup.find_all("div")
-        title = soup.find_all("title")[0].text
+        titles = soup.find_all("title")
+        if titles:
+            title = titles[0].text
+        else:
+            title = "No title found"
```

Comment:
would you prefer this over `""`?

Reply:
yes, otherwise, it will return `IndexError: list index out of range`